modules/apps/marketplace/marketplace-settings-web/src/main/java/com/liferay/marketplace/settings/web/internal/constants/MarketplaceSettingsPortletKeys.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.marketplace.settings.web.internal.constants;
+
+/**
+ * @author Keven Leone
+ */
+public class MarketplaceSettingsPortletKeys {
+
+	public static final String MARKETPLACE_SETTINGS =
```

Comment:
Is this constant needed? Can't we use `ConfigurationAdminPortletKeys.INSTANCE_SETTINGS` instead?

Reply:
Interesting, I didn't know that

I'll make the change :D